src/Components/GeneralComponents/GeneralWindow.css

```diff
@@ -1,3 +1,7 @@
+.generalWindowCloseButton {
+  margin-left: auto;
+}
```

Comment:
best option?
im sure it could be better, consult Eyal if you cant make it happen

Reply:
for now it's ok, might fix ont eh next branch along with GeneralWindow